plugins/catalog-backend-module-msgraph/src/microsoftGraph/read.ts

```diff
@@ -74,6 +75,7 @@ export async function defaultUserTransformer(
 }
 
 export async function readMicrosoftGraphUsers(
+  logger: Logger,
```

Comment:
Can we move this into the `options` parameter instead? We have a bit of a new emerging standard of sorts, where we either try to make one single large options object for complex methods, or have a small number of very obviously "core" parameters that drive the functionality, and a smaller options object with all of the auxiliary things.

Bit of a nitpick, but we've seen functions/methods like this grow and grow over time and become really unwieldy and hard to call and evolve.